### PR TITLE
Add version for ironware

### PR DESCRIPTION
**Pull request by wkamlun**
Add version for ironware

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

[ironware_ironware.zip](https://github.com/librenms/librenms/files/7577877/ironware_ironware.zip)
If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by wkamlun** [comment]
Test data
[ironware_ironware.zip](https://github.com/librenms/librenms/files/7579287/ironware_ironware.zip)

**Comment by wkamlun** [comment]
[ironware_ironware.zip](https://github.com/librenms/librenms/files/7579332/ironware_ironware.zip)

**Review by wkamlun — COMMENTED** [review]
[ironware_ironware.zip](https://github.com/librenms/librenms/files/7579284/ironware_ironware.zip)
Test data added

File: includes/definitions/discovery/ironware.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ modules:
         hardware_mib: FOUNDRY-SN-ROOT-MIB
         sysDescr_regex: '/IronWare Version V(?<version>.*) Compiled on/'
         serial: FOUNDRY-SN-AGENT-MIB::snChasSerNum.0
+        version: .1.3.6.1.4.1.1991.1.1.2.1.11.0
```
Comment (Jellyfrog) [inline includes/definitions/discovery/ironware.yaml:27]: Doesn't it have a mib name?

**Comment by Jellyfrog** [comment]
You need to commit your test data

File: includes/definitions/discovery/ironware.yaml (hunk 1 above)
Comment (bakerds) [reply includes/definitions/discovery/ironware.yaml:27]: `FOUNDRY-SN-AGENT-MIB::snAgImgVer.0` ?

Comment (murrant) [reply includes/definitions/discovery/ironware.yaml:27]: mixing numeric and non-numeric is bad, causes LibreNMS do snmptranslates on everything.

Comment (wkamlun) [reply includes/definitions/discovery/ironware.yaml]: Thanks, just updated to named OID.

**Review by Jellyfrog — APPROVED** [review]

**Comment by Jellyfrog** [comment]
Thanks (I merged it without the test data, It's not needed for such a small change)

**Pull request merged**